aggregation-worker/src/main/java/au/org/emii/aggregator/catalogue/CatalogueReader.java

```diff
@@ -0,0 +1,80 @@
+package au.org.emii.aggregator.catalogue;
```

Comment:
This will leave us will CatalogueReader code in two places. Can we something to avoid the duplication?

Reply:
Not at this stage.  Current WPS extension codebase will disappear at some point.